yarn.lock

```diff
@@ -119,27 +119,6 @@
     semver "^5.4.1"
```

Comment:
I see changes in `ui/package.json` but not in `package.json`. Because there are no changes in the root package json, any reason we have changes in the root `yarn.lock`?